### PR TITLE
feat: update hybridsql Dockerfile

### LINKED ISSUE #1124: make the compilation image version as the same version tag to the release version
1. make the compilation image version have the same version tag as the openmldb release version. currently they are not consistent (0.4.1 vs 0.4.0)
2. make the dependency in the compilation image static, bind to the corresponding openmldb version, no need to download the latest zetasql.

### PR DESCRIPTION
**Pull request by aceforeverd**
close #1124

- compile image bind to const version of zetasql, compile inside hybridsql won't download deps (usually in a release tag)
- add hybridsql docker build for aarch64

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -122,12 +122,23 @@ THIRD_PARTY_SRC_DIR ?= $(MAKEFILE_DIR)/thirdsrc
...
+#  if true, thirdparty download is skipped
+# zetasql check separately since it update more frequently:
+#   it will updated if the variable '$ZETASQL_VERSION' (defined in docker) equal to current code
 thirdparty-fast:
-	if [ $(THIRD_PARTY_DIR) != "/deps/usr" ] ; then \
+	@if [ $(THIRD_PARTY_DIR) != "/deps/usr" ] ; then \
 	    echo "fullly setup thirdparty"; \
 	    $(MAKE) thirdparty; \
+	elif [ -n "$(ZETASQL_VERSION)" ]; then \
+	    NEW_ZETASQL_VERSION=$( grep 'set(ZETASQL_VERSION' FetchZetasql.cmake | sed 's/set(ZETASQL_VERSION[[:space:]]*//g') | sed -E 's/[[:space:]]|)//g'; \
```
Comment (zhanghaohit) [inline Makefile]: FetchZetasql.cmake should be `third-party/cmake/FetchZetasql.cmake`?

Strangely, I cannot make this command work. `echo $(NEW_ZETASQL_VERSION)` always got empty.

Comment (aceforeverd) [reply Makefile]: yes, it not work

Comment (lumianph) [reply Makefile]: by design it should be empty?

Comment (aceforeverd) [reply Makefile]: `ZETASQL_VERSION` should only be set in compile image
I fixed it, pls check again

**Review by lumianph — APPROVED** [review]

**Review by zhanghaohit — APPROVED** [review]

**Pull request merged**